crates/plugins/src/install.rs

```diff
@@ -89,7 +175,7 @@ impl PluginInstaller {
         // TODO: this is unix only. Look into whether permissions are preserved
         archive.set_preserve_permissions(true);
         // Create subdirectory in plugins directory for this plugin
-        let plugin_sub_dir = self.plugins_dir.join(&self.name);
+        let plugin_sub_dir = self.plugins_dir.join(plugin_name);
```

Comment:
Why is this changed instead of `self.name`? `plugin_manifest.name` == `plugin $name` == `$name.json` right?

Reply:
The name is not required when installing from local and remote manifests as described above.